tests/actions.go

```diff
@@ -833,39 +842,239 @@ func checkoutTag(tagName string) error {
 	return nil
 }
 
-func (oa *operatorActions) DeployScheduledBackup(info *TidbClusterInfo) error {
+func (oa *operatorActions) DeployAdHocBackup(info *TidbClusterInfo) error {
+	glog.Infof("begin to deploy adhoc backup cluster[%s] namespace[%s]", info.ClusterName, info.Namespace)
+	defer func() {
+		glog.Infof("deploy adhoc backup end cluster[%s] namespace[%s]", info.ClusterName, info.Namespace)
+	}()
+	sets := map[string]string{
+		"clusterName":  info.ClusterName,
+		"name":         "test-backup",
+		"mode":         "backup",
+		"user":         "root",
+		"password":     info.Password,
+		"storage.size": "10Gi",
+	}
+	var buffer bytes.Buffer
+	for k, v := range sets {
+		set := fmt.Sprintf(" --set %s=%s", k, v)
+		_, err := buffer.WriteString(set)
+		if err != nil {
+			return err
+		}
+	}
+
+	setStr := buffer.String()
+	fullbackupName := fmt.Sprintf("%s-backup", info.ClusterName)
+	cmd := fmt.Sprintf("helm install -n %s --namespace %s /charts/%s/tidb-backup %s",
+		fullbackupName, info.Namespace, info.OperatorTag, setStr)
+	glog.Infof("install adhoc deployment [%s]", cmd)
+	res, err := exec.Command("/bin/sh", "-c", cmd).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to launch adhoc backup job: %v, %s", err, string(res))
+	}
 	return nil
 }
 
-func (oa *operatorActions) CheckScheduledBackup(info *TidbClusterInfo) error {
+func (oa *operatorActions) CheckAdHocBackup(info *TidbClusterInfo) error {
+	glog.Infof("begin to clean adhoc backup cluster[%s] namespace[%s]", info.ClusterName, info.Namespace)
+	defer func() {
+		glog.Infof("deploy clean backup end cluster[%s] namespace[%s]", info.ClusterName, info.Namespace)
+	}()
+
+	jobName := fmt.Sprintf("%s-%s", info.ClusterName, "test-backup")
+	fn := func() (bool, error) {
+		job, err := oa.kubeCli.BatchV1().Jobs(info.Namespace).Get(jobName, metav1.GetOptions{})
+		if err != nil {
+			glog.Errorf("failed to get jobs %s ,%v", jobName, err)
+			return false, nil
+		}
+		if job.Status.Succeeded == 0 {
+			glog.Error("cluster [%s] back up job is not completed, please wait! ", info.ClusterName)
+			return false, nil
+		}
+
+		return true, nil
+	}
+
+	err := wait.Poll(DefaultPollInterval, DefaultPollTimeout, fn)
+	if err != nil {
+		return fmt.Errorf("failed to launch scheduler backup job: %v", err)
+	}
 	return nil
 }
 
-func (oa *operatorActions) DeployAdHocBackup(info *TidbClusterInfo) error {
+func (oa *operatorActions) Restore(from *TidbClusterInfo, to *TidbClusterInfo) error {
+	glog.Infof("begin to deploy restore cluster[%s] namespace[%s]", from.ClusterName, from.Namespace)
+	defer func() {
+		glog.Infof("deploy restore end cluster[%s] namespace[%s]", to.ClusterName, to.Namespace)
+	}()
+	sets := map[string]string{
+		"clusterName":  to.ClusterName,
+		"name":         "test-backup",
+		"mode":         "restore",
+		"user":         "root",
+		"password":     to.Password,
+		"storage.size": "10Gi",
```

Comment:
`10Gi` may be too small for backup job

Reply:
this setting just suit for my cluster, We can modify it during stress testing。